packages/eslint-plugin/lib/config/mdx.ts

```diff
@@ -1,16 +1,19 @@
 import type { Linter } from "eslint";
-import { mdxFiles } from "../utils/patterns";
+import { concat } from "eslint-flat-config-utils";
+import * as mdxPlugin from "eslint-plugin-mdx";
+import { mdxFiles } from "../utils/patterns.ts";
 
-const config: Linter.Config = {
-    overrides: [
-        {
-            files: mdxFiles,
-            extends: ["plugin:mdx/recommended"]
-        }
-    ]
-};
+const config: Linter.FlatConfig[] = await concat(
+    {
+        name: "Workleap/MDX",
+        ...mdxPlugin.flat,
+        files: mdxFiles
+    },
+    {
+        name: "Workleap/MDX",
+        ...mdxPlugin.flatCodeBlocks,
+        files: mdxFiles
```

Comment:
Is there a reason why it requires to be define in 2 blocks?

Also, are we still getting only the "recommended" rules?

Reply:
I went back to the docs on this one and realized that code blocks is part of flat config, so there is no need to do it twice. I do need to add a processor to hit the code blocks. The recommended rules should be part of the flat config.